_pytest/vendored_packages/pluggy.py

```diff
@@ -540,7 +540,7 @@ def add_hookcall_monitoring(self, before, after):
         of HookImpl instances and the keyword arguments for the hook call.
 
         ``after(outcome, hook_name, hook_impls, kwargs)`` receives the
-        same arguments as ``before`` but also a :py:class:`_CallOutcome`` object
+        same arguments as ``before`` but also a :py:class:`_CallOutcome <_pytest.vendored_packages.pluggy._CallOutcome>` object
```

Comment:
@nicoddemus what i mean is that the pluggy details we include in the docs go wrong because we change the absolute name of everything inside pluggy due to vendoring (so the change to pluggy is correct for the context of making the pytest docs correct, but it breaks vendoring